localflavor/br/forms.py

```diff
@@ -92,26 +84,28 @@ def clean(self, value):
         value = force_text(value)
         if value == '':
             return value
-        valid_values = set([force_text(k) for k, v in self.widget.choices])
+        valid_values = set([force_text(entry[0]) for entry in self.widget.choices])
         if value not in valid_values:
             raise ValidationError(self.error_messages['invalid'])
         return value
 
 
-def DV_maker(v):
+def dv_maker(v):
```

Comment:
`def valid_nik_date(year, month, day): ` is not a public method, it is a local function inside the clean method so it is not exposed publicly